### PR TITLE
fix: after event mail separated for speaker and owner/organizers

### LINKED ISSUE #6620: After-Event Mail: Separate Email Template of Speaker and Owner/Organizer
There is an after-event message, that should not go to the speaker and organizer at the same time, which can be viewed at https://eventyay.com/admin/messages/list .

Screenshot from 2019-09-16 10-59-01

Instead there should be
a) a message to the owner and organizers
b) a message to the speaker
These message should be differently and don't include upcoming events.

1. Please implement a message to the "Owner, Organizer"
Subject: {eventname} is over. Thank you for using Eventyay!
Email: Hi {email}, you event {eventname] is over. Thank you for using the Eventyay platform for your event. We hope you enjoyed it. 

2. Implement a message only for accepted speakers
Subject: {eventname} is over. Thank you for participating!
Email: Hi {email},thank you for participating in our {eventname]. We hope you enjoyed it and keep in touch.

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6620 

#### Short description of what this resolves:
after event mail seperated for speaker and owner/organizers

#### Checklist

-  [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal Please check

**Comment by codedsun** [comment]
@iamareebjamal chck this

**Comment by codedsun** [comment]
@iamareebjamal Review this!

**Comment by iamareebjamal** [comment]
No need to ping again and again. Saw it the first time. We're volunteers here and schedule time accordingly to the priority of issues and PRs. Ask peers to review before maintainers. I only have a single brain, 2 hands and 2 eyes, can't manage 10 repos withtin milliseconds of sending the PR. With no peer reviews, PRs may not get merged for even a month if they are not as important.

**Comment by codedsun** [comment]
Okay, sorry for that.Thanks np!

**Comment by codedsun** [comment]
@prateekj117 Please review

**Comment by codedsun** [comment]
@kushthedude @prateekj117 Please review!

**Comment by codedsun** [comment]
@prateekj117 If time allows, please review. Thanks

File: app/api/helpers/system_mails.py [hunk 1]
```diff
@@ -58,12 +58,21 @@
...
     },
     AFTER_EVENT: {
-        'recipient': 'Owner, Organizer, Speaker',
-        'subject': u'Event {event_name} is over',
+        'recipient': 'Owner, Organizer',
+        'subject': u'Event {event_name} is over. Thank you for using Eventyay!',
+        'message': (
+            u"Hi {email},<br/>" +
+            u"your event {event_name} is over. We hope you enjoyed it. "
+            u"Thank you for using the Eventyay platform for your event. We hope you enjoyed it.<br />"
```
Comment (iamareebjamal) [inline app/api/helpers/system_mails.py]: Eventyay?

Comment (codedsun) [reply app/api/helpers/system_mails.py]: Fixed ^_^

**Comment by codedsun** [comment]
FIxed, check now!

File: app/api/helpers/system_mails.py [hunk 2]
```diff
@@ -58,12 +58,21 @@
         )
     },
     AFTER_EVENT: {
-        'recipient': 'Owner, Organizer, Speaker',
-        'subject': u'Event {event_name} is over',
+        'recipient': 'Owner, Organizer',
+        'subject': u'Event {event_name} is over. Thank you for using Eventyay!',
```
Comment (iamareebjamal) [inline app/api/helpers/system_mails.py]: ?

Comment (codedsun) [reply app/api/helpers/system_mails.py]: omg ;( Apologies

**Comment by iamareebjamal** [comment]
That that `{frontend}/admin/messages/list` works according to new changes

**Comment by codedsun** [comment]
> That that `{frontend}/admin/messages/list` works according to new changes

??

**Comment by codedsun** [comment]
@iamareebjamal I need to insert this record to the DB, how do we do that? Like I shall give you the query ?

**Comment by codedsun** [comment]
How do i insert a record in messgage_Settings for the same?

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- app/api/helpers/system_mails.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4615073&bid=15479197)

**Comment by iamareebjamal** [comment]
By creating a migration

**Comment by codedsun** [comment]
@iamareebjamal Created the migration please check, also the frontend functionality is a little broken. It says that we are showing all the items, but the api has either the pagination functionality and only 20 items are returned in one go. 

image



But the API is working fine

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- app/api/helpers/system_mails.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4615073&bid=15479197)

**Comment by iamareebjamal** [comment]
Must be a bug in pagination. Raise an issue there

File: migrations/versions/rev-2019-12-09-00:14:14-de295fee375b_insert_after_event_setting_for_speaker.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+def upgrade():
+    op.execute('INSERT into message_settings (action,mail_status,notification_status,user_control_status) VALUES '
+               '(\'After Event to Speaker\', true, true, true )')
+    pass
+
+
+def downgrade():
+    pass
```
Comment (iamareebjamal) [inline migrations/versions/rev-2019-12-09-00:14:14-de295fee375b_insert_after_event_setting_for_speaker.py]: Add downgrade command as well

File: migrations/versions/rev-2019-12-09-00:14:14-de295fee375b_insert_after_event_setting_for_speaker.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+# revision identifiers, used by Alembic.
+revision = 'de295fee375b'
+down_revision = '0223c881d135'
+
+
+def upgrade():
+    op.execute('INSERT into message_settings (action,mail_status,notification_status,user_control_status) VALUES '
+               '(\'After Event to Speaker\', true, true, true )')
+    pass
```
Comment (iamareebjamal) [inline migrations/versions/rev-2019-12-09-00:14:14-de295fee375b_insert_after_event_setting_for_speaker.py]: remove

Comment (codedsun) [reply migrations/versions/rev-2019-12-09-00:14:14-de295fee375b_insert_after_event_setting_for_speaker.py]: removed the `pass`

**Comment by codedsun** [comment]
Check now. I have raised an issue also https://github.com/fossasia/open-event-frontend/issues/3693

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- app/api/helpers/system_mails.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4615073&bid=15479197)

**Comment by codedsun** [comment]
Also, please tell if their is something pending here.?

**Comment by iamareebjamal** [comment]
Need to think about previous message being not present in the system.

**Comment by codedsun** [comment]
>Need to think about previous message being not present in the migrations.

Because they are added once you run populate_db. Is this a task for me or your end??

**Comment by iamareebjamal** [comment]
> Because they are added once you run populate_db

Not if it has been run once before

> Is this a task for me or your end??

Whoever thinks of the solution first

**Comment by codedsun** [comment]
@mrsaicharan1 - Can you help me here?
>Need to think about previous message being not present in the migrations.

**Pull request closed without merging**